### ISSUE TITLE
Remove/replace old hotkeys-as-keybindings system

### ISSUE THREAD

**Issue by PeridexisErrant** [DFHack/dfhack#731]
It made sense when DFHack was young and didn't have the keybinding command.

These days, it's cruft and with the number of commands available it's increasingly likely to be triggered by accident, by someone who names their burrow something like "points", "lever", "source", "liquids", autofarm"...

We just don't need it and keeping such an old feature around isn't helping anyone.

**Comment by warmist** [comment]
Yes :)

**Comment by PeridexisErrant** [comment]
Whoever takes out this code should also remove the note about it in the docs [here}(https://dfhack.readthedocs.org/en/stable/docs/Core.html#miscellaneous-notes).

**Comment by lethosor** [comment]
Yeah, I tried doing this at one point, but I ran into a bunch of older code that was more complicated to remove.
From what I can tell, bindings like F1@dwarfmode should be equivalent, so we might as well get rid of the old system.

**Comment by lethosor** [comment]
I did run into someone using this (intentionally) on [Reddit](https://www.reddit.com/r/dwarffortress/comments/65mb9g/biweekly_df_questions_thread/dgfdt8m/), which was the inspiration for `tweak hotkey-clear`. Maybe removing this feature entirely isn't the way to go, since it is easy to configure in-game. Some sort of additional hotkey mode (besides the built-in None/Zoom/Unit/Item) could work.

**Comment by lethosor** [comment]
It looks like this is one of the only ways to set up keybindings in PRINT_MODE:TEXT (`keybinding` doesn't work there). However, it currently doesn't work on OS X. Moving this to a plugin would probably be more reliable (although I'm not sure if the hotkey thread is still necessary, so it might not be trivial).

**Comment by warmist** [comment]
I vote we fix keybindings in PRINT_MODE:TEXT

**Comment by ab9rf** [comment]
On review, this issue appears completed/redundant. Even if DF does someday regain ncurses text mode (which I think is unlikely), it probably won't do so in a way that makes this directly relevant again.

**Comment by lethosor** [comment]
This wasn't specific to ncurses mode, but it looks like our integration with the DF [hotkeys](https://dwarffortresswiki.org/index.php/Hotkeys) feature in question was disabled in https://github.com/DFHack/dfhack/commit/218fbdbd452a8c88636448bc16c0454c3d2906c5#diff-e2be5c8705bcc85f17d830f04f92817ed2da657aa4d75ea3774481bf30b37a80R2269-R2457
